system/netdata.service.in

```diff
@@ -63,6 +63,8 @@ CapabilityBoundingSet=CAP_SYS_RESOURCE
 CapabilityBoundingSet=CAP_NET_RAW
 # is required for cgroups plugin
 CapabilityBoundingSet=CAP_SYS_CHROOT
+# is required for samba plugin
```

Comment:
These changes required for all collectors that use `sudo` (bash and python).

Reply:
True. I'll edit the comment and commit message accordingly.